File: SISMUEBLUM JAVA/Local.java
import java.util.*;
public class Local{
	private int local_id;
	private String local_direccion;
	private int local_telef;
	private ArrayList<Vendedor> local_vendedores;

	public Local(int local_id, string local_direccion, int local_telef){
		this.local_id = local_id;
		this.local_direccion = local_direccion;
        	this.local_telef = local_telef;
		local_vendedores = new List<Vendedor>();
		
	}
	
	public Local(){
		local_vendedores = new List<Vendedor>();
	}
	
	public Local(){
		local_vendedores = new ArrayList<Vendedor>();
	}

	public int getLocal_id(){
		return local_id;
	}
	
	public void setLocal_id(int local_id){
		this.local_id = local_id;
	}
	
	public String getLocal_direccion(){
		return local_direccion;
	}
	
	public void setLocal_direccion(String local_direccion){
		this.local_direccion = local_direccion;
	}

	public List<Vendedor> getLocal_vendedores(){
		return local_vendedores;
	}
	
	public void setLocal_vendedores(List<Vendedor> local_vendedores)
		this.local_vendedores = local_vendedores;
	}
	
	public int getLocal_telef(){
		return local_telef;
	}
	
	public void setLocal_telef(int local_telef){
		this.local_telef = local_telef;
	}


	public void agregarVendedor(Vendedor v){
		local_vendedores.Add(v);
	}

	public String consultarVendedores(){
	}

}

File: SISMUEBLUM C#/Local.cs
using System;
public class Local{
	private int _id;
	private string _direccion;
	private int _telef;
	private List<Vendedor> _vendedores;

	public Local(int _id, string _direccion, int _telef){
		this._id = _id;
		this._direccion = _direccion;
        	this._telef = _telef;
		_vendedores = new List<Vendedor>();
		
	}
	
	public Local(){
		_vendedores = new List<Vendedor>();
	}
	
	~Local(){
	}

	public int local_id{
		set{
			this._id = value;
		}
		get{
			return _id;
		}
	}

	public string local_direccion{
		set{
			this._direccion = value;
		}
		get{
			return _direccion;
		}
	}
	
	public int local_telef{
		set{
			this._telef = value;
		}
		get{
			return _telef;
		}
	}

	public List<Vendedor> local_vendedores{
		set{
			this._vendedores = value;
		}
		get{
			return _vendedores;
		}
	}

	public void agregarVendedor(Vendedor v){
		_vendedores.Add(v);
	}

	public string consultarVendedores(){
    		string cadena = "";
		return cadena;
	}

}
